botcore/utils/logging.py

```diff
@@ -0,0 +1,45 @@
+"""Custom :obj:`logging.Logger` class that implements a new ``"TRACE"`` level."""
```

Comment:
This is fine for now, but it wouldn't hurt to make it more generic now. Up to you.